### ISSUE TITLE
main.CRITICAL: Broken reference:

### ISSUE THREAD

**Issue by rbncha** [magento/magento2#3900]
I am trying to install fresh Magento 2.0 to my centOS 6.5, php 5.6.19 via composer + browser install wizard. I have followed all the steps and install is success. All file permission are modified as suggested. My apache server is running with nobody:nobody user. I keep getting this below error. I tried several complete reinstalls with/without data. Same error.

Can anyone please help me get out of this issue ?

NOTE: A localhost similar install process in php 7.0.1. is a success with even the data import using data import module. Could it be due to nobody user or maybe php ver. 5.6.19 ? Your hints and suggestions is highly appreciated.

```
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'global_notices' element cannot be added as child to 'global.notices', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'logo' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'global.search' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'user' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'menu' element cannot be added as child to 'page.menu', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'breadcrumbs' element cannot be added as child to 'page.breadcrumbs', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'formkey' element cannot be added as child to 'page.formkey', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'page.title' element cannot be added as child to 'main.top', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'messages' element cannot be added as child to 'page.messages', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'page.actions.toolbar' element cannot be added as child to 'page.main.actions', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'legal' element cannot be added as child to 'footer', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'system_messages' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'unread_system_messages' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'notification_window' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'notification.messages' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'header.inner.left' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'header.inner.right' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'logo' tries to reorder itself towards '', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'global.search' tries to reorder itself towards 'logo', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'user' tries to reorder itself towards '', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'page.actions.toolbar' tries to reorder itself towards '', but their parents are different: 'page.main.actions' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'system_messages' tries to reorder itself towards '', but their parents are different: 'notifications' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'unread_system_messages' tries to reorder itself towards 'system_messages', but their parents are different: 'notifications' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'header.inner.left' tries to reorder itself towards '', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'header.inner.right' tries to reorder itself towards 'header.inner.left', but their parents are different: 'header' and '' respectively. [] []
[2016-03-26 17:49:06] main.CRITICAL: Broken reference: the 'global.search' tries to reorder itself towards 'notification.messages', but their parents are different: 'header.inner.right' and '' respectively. [] []

```

**Comment by mazhalai** [comment]
@rbncha These errors dont tell us much. What is your exact issue? Are you unable to install? 
Also, have you checked if your file permissions are as recommended [here](http://devdocs.magento.com/guides/v2.0/install-gde/prereq/apache-user.html) and [here](http://devdocs.magento.com/guides/v2.0/install-gde/prereq/file-system-perms.html)?

**Comment by piotrekkaminski** [comment]
Possibly related MAGETWO-53410

**Comment by piotrekkaminski** [comment]
The issue is fixed internally and will be delivered in 2.1

**Comment by AsadDhothar** [comment]
Got same issue in magento 2.2.7 
```
`[2019-04-18 09:56:18] main.INFO: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header.inner.right' and 'header' respectively. [] []
[2019-04-18 09:57:09] main.INFO: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header.inner.right' and 'header' respectively. [] []
[2019-04-18 09:57:12] main.INFO: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header.inner.right' and 'header' respectively. [] []
[2019-04-18 09:57:15] main.INFO: Broken reference: the 'notification.messages' tries to reorder itself towards 'user', but their parents are different: 'header.inner.right' and 'header' respectively. [] []`
```
